CMakeLists.txt

```diff
@@ -123,7 +123,7 @@ add_custom_command(
 )
 
 install(
-  FILES LICENSE README.md THIRD_PARTY.md
```

Comment:
README.md should be included too into package.

Reply:
README contains mostly build instructions and detailed usage, do you think it makes sense to include it as-is in the pre-built package? I wouldnt expect to find build instructions in a package that does not contain the source

Reply:
Probably we can move building instructions to different file. Main idea is that README.md has all info that user should read first (like project description, license information, contact information), and then go to USAGE.md for actual settings.

Reply:
That makes sense, I'll work towards that. Our current full usage instructions make use of HTML formatting that is only displayed correctly on Github, so that'll have to change to be shipped